memex/src/test/java/com/johnlpage/memex/cucumber/steps/BaseCucumberSteps.java

```diff
@@ -0,0 +1,17 @@
+package com.johnlpage.memex.cucumber.steps;
+
+import org.springframework.beans.factory.annotation.Value;
+
+public class BaseCucumberSteps {
```

Comment:
File name says `Steps`, but this file cant have any steps as we are unable to extend files with `Steps`

Reply:
It will be mostly used for common helper functions.
Open to suggestions on file naming

Reply:
Done